robyn/reloader.py

```diff
@@ -115,11 +117,19 @@ def reload(self):
         clean_rust_binaries(self.built_rust_binaries)
         self.built_rust_binaries = compile_rust_files(self.directory_path)
 
-        self.process = subprocess.Popen(
-            [sys.executable, *arguments],
-            env=new_env,
-            start_new_session=False,
-        )
+        if self.config.dev and self.config.file_path is not None:
+            subprocess.call(f'cd {"/".join(self.config.file_path.split("/")[-2:-1])}', shell=True)
+            self.process = subprocess.Popen(
+                [sys.executable, "-m", self.config.file_path.split("/")[-2], *arguments],
```

Comment:
why do we need a `-m` here?

Reply:
to run again as a module whilst reloading...? since, we have run the app as a module initially like `python3 -m src --dev`